db/seeds/CBDU17.json

```diff
@@ -0,0 +1,65 @@
+{
+  "date_flag": "expired",
+  "tier": "UPPER",
+  "registrationType": "carrier_broker_dealer",
+  "businessType": "limitedCompany",
+  "otherBusinesses": "yes",
+  "isMainService": "yes",
+  "onlyAMF": "no",
+  "companyName": "Limited Company Seed",
+  "companyNo": "946107",
+  "firstName": "Test",
+  "lastName": "User",
+  "phoneNumber": "01234 567890",
+  "contactEmail": "user@waste-exemplar.gov.uk",
+  "addresses": [{
+      "addressType": "REGISTERED",
+      "addressMode": "manual-uk",
+      "houseNumber": "Unit 5",
+      "addressLine1": "Horizon House",
+      "addressLine2": "Deanery Road",
+      "townCity": "Bristol",
+      "postcode": "BS1 5AH",
+      "location": {
+        "lat": 0,
+        "lon": 0
+      }
+    },
+    {
+      "addressType": "POSTAL",
+      "houseNumber": "Richard Fairclough House",
+      "addressLine1": "Knutsford Road",
+      "addressLine2": "Latchford",
+      "addressLine3": "",
+      "addressLine4": "",
+      "townCity": "Warrington",
+      "postcode": "WA4 1HT",
+      "country": ""
+    }
+  ],
+  "keyPeople": [{
+    "firstName": "Test",
+    "lastName": "Employee",
+    "dateOfBirth": "1978-01-01",
+    "position": "Director",
+    "personType": "KEY",
+    "convictionSearchResult": {
+      "matchResult": "NO",
+      "confirmed": "no"
+    }
+  }],
+  "accountEmail": "user@waste-exemplar.gov.uk",
+  "declaredConvictions": "no",
+  "declaration": 1,
+  "regIdentifier": "CBDU17",
+  "metaData": {
+    "anotherString": "userDetailAddedAtRegistration",
+    "status": "ACTIVE",
```

Comment:
Should the status be `EXPIRED` here or are we just testing the date? Ditto for the other expired seed.

Reply:
Good spot, I've updated